aikau/src/main/resources/alfresco/forms/Form.js

```diff
@@ -399,6 +399,10 @@ define(["dojo/_base/declare",
                if (widget && widget.config)
                {
                   widget.config.showValidationErrorsImmediately = this.showValidationErrorsImmediately;
+                  if (widget.config.pubSubScope)
+                  {
+                     this.alfLog("warn", "It is not recommended to set a pubSubScope attribute on a form control nested within a form - the pubSubScope should be inherited!", this, widget);
```

Comment:
Why is it not recommended? No mention that the field won't be submitted anywhere?